src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  let personPartner = 'I am not married';
+
+  if (isMarried) {
+    if (sex === 'f') {
+      personPartner = `${partnerName} is my husband`;
+    } else {
+      personPartner = `${partnerName} is my wife`;
+    }
+  }
```

Comment:
- It's better to create a separate function to get this text to avoid additional logic in the component
- 'f' is a magic value
- You repeat logic in "personPartner = `${partnerName} is my husband`"

You can simplify like this
```
const personPartnerText = isMarried ?  `${partnerName} is my ${sex === SEX.female ? 'husband' : 'wife'}` : DEFAULT_PERSON_PARTNER_TEXT;
```